### DISCUSSION TITLE
"Contact Us" links don't trigger navigation on paste.twilio.design

### DISCUSSION THREAD

**Discussion by hughrawlinson** [twilio-labs/paste#2627] [General] [answered]
Hi! I tried in Firefox and Chrome, and the three links under the "contact us" menu at the bottom left of paste.twilio.design don't go anywhere. It looks like they render as `a` tags without any `href`. Maybe you need to add a `passHref` attribute to the `Link`?

**Comment by shleewhite** [comment]
Hi @hughrawlinson 

Thanks for letting us know! We will get a fix up for that ASAP.

**Reply by shleewhite** [reply]
@hughrawlinson There is a PR for the fix now: https://github.com/twilio-labs/paste/pull/2634

I will post again when it is merged!

**Comment by SiTaggart** [comment, marked as answer]
Hi @hughrawlinson 

This fix to Menu Items and our own Doc site is now live and in the wild on NPM.

Thanks for calling that out!
